lib/vector-core/src/schema/definition.rs

```diff
@@ -452,13 +452,120 @@ impl Definition {
     }
 }
 
+#[cfg(any(test, feature = "test"))]
+mod test_utils {
+    use super::*;
+    use crate::event::{Event, LogEvent};
+
+    impl Definition {
+        /// Asserts that the schema definition is _valid_ for the given event.
+        /// This will panic if the definition is not valid.
```

Comment:
Doesn't look like it panics?

Reply:
I guess this should be down with the `assert` fn?